qiskit/chemistry/results/electronic_structure_result.py

```diff
@@ -269,40 +283,68 @@ def formatted(self) -> List[str]:
             lines.append('~ Nuclear repulsion energy (Hartree): {}'.
                          format(round(self.nuclear_repulsion_energy, 12)))
             lines.append('> Total ground state energy (Hartree): {}'.
-                         format(round(self.energy, 12)))
+                         format(round(self.total_energies[0], 12)))
+
+        if len(self.computed_energies) > 1:
+            lines.append(' ')
+            lines.append('=== EXCITED STATE ENERGIES ===')
+            lines.append(' ')
+            for idx, (elec_energy, total_energy) in enumerate(zip(self.electronic_energies[1:],
+                                                                  self.total_energies[1:])):
+                lines.append('{: 3d}: '.format(idx+1))
+                lines.append('* Electronic excited state energy (Hartree): {}'.
+                             format(round(elec_energy, 12)))
+                lines.append('> Total excited state energy (Hartree): {}'.
+                             format(round(total_energy, 12)))
+
         if self.has_observables():
-            line = '  Measured::'
-            if self.num_particles is not None:
-                line += ' # Particles: {:.3f}'.format(self.num_particles)
-            if self.spin is not None:
-                line += ' S: {:.3f}'.format(self.spin)
-            if self.total_angular_momentum is not None:
-                line += ' S^2: {:.3f}'.format(self.total_angular_momentum)
-            if self.magnetization is not None:
-                line += ' M: {:.5f}'.format(self.magnetization)
-            lines.append(line)
+            lines.append(' ')
```

Comment:
In the past we had the ground state and then a list of excited states with energies and observables in one line. I just wonder if this is printed out in blocks just how user friendly it is since you more have to correlate in the each list for all the information of each state - in the past the information was together for one state, so you just went from one to next and had all the pertinent information

Reply:
today we went for this formatting. We can for sure change the formatting in the future, but in the sake of time I would advise not to do this now.

Reply:
I added a TODO to the Backlog which we keep offline (the qiskit naming discussion box note)